### PR TITLE
Bug/800 user tag updates

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

**Comment by ciaranschutte** [comment]
@hlminh2000 few additional fixes here when you get a chance please :)

File: package.json [hunk 1]
```diff
@@ -25,12 +25,12 @@
     "md5": "^2.2.1",
     "popback": "^0.0.3",
     "query-string": "^5.1.0",
-    "react": "^16.3.0",
+    "react": "^16.5.2",
     "react-app-rewire-hot-loader": "^1.0.1",
     "react-autocomplete": "^1.8.1",
     "react-component-component": "^1.2.1",
     "react-copy-to-clipboard": "^5.0.1",
-    "react-dom": "^16.3.0",
```
Comment (hlminh2000) [inline package.json]: can we close the other PR then?

File: src/common/displayFormatters.js [hunk 2]
```diff
@@ -23,3 +23,15 @@ export const formatPhoneNumber = phone => {
...
+ * Title cases string
+ * @param {String} string
+ * @returns {String}
+ */
+export const titleCase = (string = '') => {
+  return string.replace(
+    /\w\S*/g,
+    text => text.charAt(0).toUpperCase() + text.substr(1).toLowerCase(),
+  );
+};
```
Comment (hlminh2000) [inline src/common/displayFormatters.js]: lodash's `startCase`? :)))

File: package.json (hunk 1 above)
Comment (ciaranschutte) [reply package.json]: merged other pr into next

File: src/common/displayFormatters.js (hunk 2 above)
Comment (ciaranschutte) [reply src/common/displayFormatters.js]: let's leave this one as is. 
startCase would incorrectly remove hyphens from some terms eg. 'anti-inflammatory'

**Pull request merged**